### PR TITLE
Add SSL support to mesos configuration

### PR DESCRIPTION
**Pull request by veger**
This PR adds SSL-related [configuration settings](http://mesos.apache.org/documentation/latest/configuration/) in _/etc/default/mesos_ when `mesos_ssl_enabled` is true.

### CODE REVIEW

**Comment by veger** [comment]
CentOS 7 task _Downloading and enable the EPEL repository definitions._ fails

File: templates/conf-mesos.j2 [hunk 1]
```diff
@@ -8,6 +8,15 @@ LOGS={{mesos_log_location}}
 ULIMIT="{{mesos_ulimit}}"
 ZK="{{ mesos_zookeeper_masters }}"
 
+{% if mesos_ssl_enabled %}
+export LIBPROCESS_SSL_ENABLED=true
```
Comment (ernestas-poskus) [inline templates/conf-mesos.j2]: shouldn't this be prefixed with MESOS_ ?

Comment (veger) [inline templates/conf-mesos.j2]: I have it currently working using exactly this.
If you like I can check tomorrow if the MESOS_ prefix also works?

Comment (veger) [inline templates/conf-mesos.j2]: I tried `MESOS_LIBPROCESS_SSL_ENABLED` and `MESOS_SSL_ENABLED`, both do not work

Comment (ernestas-poskus) [inline templates/conf-mesos.j2]: thats odd but ok, will check why build fails

**Comment by ernestas-poskus** [comment]
Rebase on master, fixed in https://github.com/AnsibleShipyard/ansible-mesos/pull/78

**Comment by ernestas-poskus** [comment]
Thank you for contribution :+1:

**Pull request merged**